src/docs/index.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: article
+---
+<h1 class="main-header">Brainly style guide</h1>
+<h2 class="article-header  article-header--small">
+  Version {{ site.data.package.version }}
+</h2>
+
+<p class="mint-text">Welcome to the official Brainly style guide! This style guide is being used across all language versions of the Brainly website (e.g. <a href="http://brainly.com" class="mint-link">brainly.com</a>, <a href="http://znanija.com" class="mint-link">znanija.com</a>, <a href="http://nosdevoirs.fr/" class="mint-link">nosdevoirs.fr</a>), on our corporate website (<a href="http://brainly.co" class="mint-link">brainly.co</a>) and some smaller sites (e.g. landing pages).</p>
+
+<h2 class="article-header article-header--small">
+  How to use it?
+</h2>
+
+<p class="mint-text">In order to use style guide components, you have to include CSS file in the <code>&lt;head&gt;</code> of your website:</p>
+
+<pre>
+  &lt;link href="http://styleguide.brainly.com/{{ site.data.package.version }}/style-guide.css" rel="stylesheet"/&gt;
+</pre>
+
+<p class="mint-text">In order to use basic set of <a href="./basics.html#icons" class="mint-link">icons</a> include this script before the closing &lt;/body&gt; tag:</p>
+
+<pre>
+  &lt;script src="<script>document.write(document.querySelector('script[src*=subjects-icons-]').src)</script>"&gt;&lt;/script&gt;
```

Comment:
I never thought I'll use `document.write`, but here we are. Why? Note that icon files and image files are fingerprinted (e.g. `subjects-icons-e9967e3fb1.js`) and I need to expose a vaild URL in the doc, so... I'm taking this URL from the `<head>` and putting it here - tada :fireworks: